SimpleFeed/spec/controllers/post_controller_spec.rb

```diff
@@ -90,10 +106,48 @@
   end
 
   describe '#update' do
-    skip
+    let!(:post) do 
+      create(:post, title: 'Old title', name: 'Old name')
+    end
+
+    it 'changes post attributes' do
+      put :update, id: post.id, post: { title: 'New title', name: 'New name' }
+      post.reload
+      expect(post.title).to eq 'New title'
+      expect(post.name).to eq 'New name'
+    end
+
+    it 'does not accept title less than 4 chars' do
+      put :update, id: post.id, post: { title: 'abc', name: 'nil' }
+      post.reload
+      expect(post.title).not_to eq 'abc'
+    end
+
+    it 'redirects to updated post' do
+      put :update, id: post.id, post: { title: 'New name', name: 'New name' }
+      expect(response).to redirect_to(:post)
+    end
+
+    it 're-renders edit method' do
+      put :update, id: post.id, post: { title: nil, name: nil }
+      expect(response).to render_template(:edit)
+    end
   end
 
   describe '#delete' do
```

Comment:
For each method test, it might also be a good idea to include several [contexts](http://betterspecs.org/#contexts) describing the different situations that may occur within the method and the groups of tests you'd like to perform for that situation. For this method, two example contexts could be "given a valid post id" and "given an invalid post id".

I almost always include contexts in my tests, as it also helps organize test code. If you have good contexts, it'll be easy to know where to add tests later, figure out what tests you need to add for a particular context, and extract out common test code into [shared examples](http://betterspecs.org/#sharedexamples), and so on.